R/extract.R

```diff
@@ -0,0 +1,47 @@
+#' @export
+extract_parameter_set_dials.tailor <- function(x, ...) {
+  all_args <- generics::tunable(x)
+  tuning_param <- generics::tune_args(x)
+  res <-
+    dplyr::inner_join(
+      tuning_param %>% dplyr::select(-tunable),
+      all_args,
+      by = c("name", "source", "component", "component_id")
+    ) %>%
+    dplyr::mutate(object = purrr::map(call_info, eval_call_info))
+
+  dials::parameters_constr(
+    res$name,
+    res$id,
+    res$source,
+    res$component,
+    res$component_id,
+    res$object
+  )
+}
+
+eval_call_info <- function(x) {
```

Comment:
Should we make a small standalone file? If not, add some notes/link to original and `noocv`? 

(same for other replicas below)